<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>421</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Method &amp; Parameters</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images/icons.qrc" >
    <normaloff>:/images/meta.png</normaloff>:/images/meta.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="text" >
        <string>analysis method:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="method_cbo_box" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="parameter_grp_box" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>40</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Verdana</family>
      </font>
     </property>
     <property name="title" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/icons.qrc" />
 </resources>
 <connections/>
</ui>
